### DISCUSSION TITLE
出来上市

### DISCUSSION THREAD

**Discussion by LunchWarfare** [share121/inter-knot#402] [其他]
![1720334215611](https://github.com/user-attachments/assets/34658e1f-8ce5-4665-be68-df7cbf0c6426)

**Comment by paul-way52** [comment]
让西方的朋友，做！出！回！答！！！😡

**Comment by Kamisato100** [comment]
好，把他们上市！

**Comment by fireflys26710** [comment]
What was your mission in Shanghai？！

**Comment by KUNKUN4** [comment]
把bvvd上市！
